src/features/activity/ActivityDetailsScreen.tsx

```diff
@@ -0,0 +1,297 @@
+
+      // capitalize first letter of description
+      return `${customDescription
+        .charAt(0)
+        .toUpperCase()}${customDescription.slice(1)}`
+    }
+
+    if (confirmedSig.err) {
+      return 'Error'
+    }
+
+    return 'Transaction Successful'
+  }, [transaction])
+
+  const handleOpenExplorer = useCallback(async () => {
+    const url = `https://explorer.solana.com/tx/${transaction.signature}?cluster=${cluster}`
+    // Open url in browser
+    await Linking.openURL(url)
+  }, [cluster, transaction.signature])
+
+  const toggleActionSheet = useCallback(
+    (open) => () => {
+      setOptionsOpen(open)
+    },
+    [],
+  )
+
+  const showToast = useCallback(() => {
+    if (!selectedAddress) return
+    Toast.show(
+      t('generic.copied', {
+        target: ellipsizeAddress(selectedAddress),
+      }),
+    )
+  }, [selectedAddress, t])
+
+  const handleCopyAddress = useCallback(() => {
+    if (!selectedAddress) return
+
+    Clipboard.setString(selectedAddress)
+    showToast()
+    triggerNavHaptic()
+    setOptionsOpen(false)
+  }, [selectedAddress, showToast, triggerNavHaptic])
+
+  const accountOptions = useCallback(
+    () => (
+      <>
+        <ListItem
+          key="copyAddress"
+          title={t('settings.sections.account.copyAddress')}
+          onPress={handleCopyAddress}
+          selected={false}
+        />
+      </>
+    ),
+    [handleCopyAddress, t],
+  )
+
+  return (
+    <Animated.View entering={DelayedFadeIn} style={globalStyles.container}>
+      <ScrollView
+        contentContainerStyle={{ flexGrow: 1, justifyContent: 'center' }}
+      >
+        <BackScreen title={t('activityScreen.activityDetails')}>
+          <Box alignItems="center" justifyContent="center" height="100%">
+            {activityImage}
+            <Text variant="h2" marginTop="m" marginBottom="s">
+              {title}
+            </Text>
+            <Text
+              variant="body0"
+              marginBottom="s"
+              textAlign="center"
+              opacity={0.8}
+            >
+              {description}
+            </Text>
+            <Text variant="body2" textAlign="center" color="secondaryText">
+              {dateLabel}
+            </Text>
+            {AccountAddressListItems}
+            <Box width="100%">
+              <ButtonPressable
+                marginTop="xl"
+                marginHorizontal="m"
+                flexGrow={1}
+                borderRadius="round"
+                backgroundColor="white"
+                backgroundColorOpacity={1}
+                backgroundColorOpacityPressed={0.05}
+                titleColorDisabled="grey600"
+                backgroundColorDisabled="white"
+                backgroundColorDisabledOpacity={0.1}
+                titleColorPressedOpacity={0.3}
+                title={t('activityScreen.viewOnExplorer')}
+                titleColor="black"
+                onPress={handleOpenExplorer}
+                fontWeight="bold"
```

Comment:
I think adding `variant` to `ButtonPressable` would be a good idea instead of `fontWeight`

Reply:
I ended up removing the font weight per Petes feedback 👍🏽